resources/lang/en/admin/settings/general.php

```diff
@@ -2,6 +2,7 @@
 
 return [
     'ad'				        => 'Active Directory',
+    'update_null_audits'        => 'Also update audit dates on assets without a current next audit date set based on purchase date (if present)',
```

Comment:
It looks like this part isn’t built yet - but I suppose it’s fine to introduce a new string for something that we might want to make eventually.